test/validation/dry_validation_test.rb

```diff
@@ -95,7 +97,7 @@ class AlbumForm < Reform::Form
         end
       end
 
-      validation :default do
+      validation :default, with: { title: -> { title }, guitar: Guitar } do
```

Comment:
Is that a dry-v feature? And where do you need this? Isn't that all nicely available via the `form` instance?